src/Kernel/DoItVariable.class.st

```diff
@@ -179,6 +179,31 @@ DoItVariable >> key [
 	^self name
 ]
 
+{ #category : #'read/write usage' }
+DoItVariable >> markEscapingRead [
+	actualVariable markEscapingRead
+]
```

Comment:
I don't think it is the right thing to do.
When DoItVariable represents an external temp the actual temp is already analyzed for escape operations. Therefore a compilation of doIts should not change it.
I think the correct approach is to do nothing in these methods. Leave them empty.